airflow/api/common/experimental/get_dag_runs.py

```diff
@@ -50,6 +54,4 @@ def get_dag_runs(dag_id, state=None):
                            run.start_date.isoformat()),
             'dag_run_url': url_for('Airflow.graph', dag_id=run.dag_id,
                                    execution_date=run.execution_date)
-        })
-
-    return dag_runs
```

Comment:
The change in this file serves no purpose and just confuses git history. Please revert this.